src/test/java/com/stripe/functional/ServiceProductTest.java

```diff
@@ -0,0 +1,36 @@
+package com.stripe.functional;
+
+import com.stripe.BaseStripeFunctionalTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.*;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ServiceProductTest extends BaseStripeFunctionalTest {
+    @Test
+    public void testProductCreate() throws StripeException {
+        Product product = Product.create(defaultServiceProductParams);
```

Comment:
In newer tests we usually mock the API instead of sending real requests.

Not necessarily a big issue since we're going to port the test suite to use stripe-mock eventually, so just an FYI -- feel free to ignore :)